src/main/java/com/dev/cinema/config/AppConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.dev.cinema.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.dev.cinema.service",
+        "com.dev.cinema.dao"
```

Comment:
You use AuthenticationService so you need to add also "com.dev.cinema.security" here.

Reply:
My Auth Service located in the common service package along with its implementation